pkg/stub/handler.go

```diff
@@ -626,8 +623,14 @@ func (h *Handler) Handle(event v1.Event) error {
 				cfg.ConditionTrue(v1.SamplesExist) &&
 				cfg.ConditionFalse(v1.ImageChangesInProgress) &&
 				v1.GitVersionString() == cfg.Status.Version {
-				logrus.Debugf("Handle ignoring because config the same and exists is true, in progress false, and version correct")
-				return nil
+				logrus.Debugf("At steady state: config the same and exists is true, in progress false, and version correct")
+
+				// in case this is a bring up after initial install, we take a pass
+				// and see if any samples were deleted while samples operator was down
+				h.buildFileMaps(cfg)
+				h.buildSkipFilters(cfg)
```

Comment:
didn't the skipfilters already get built up on line 606?

Reply:
yep missed that ... removing this call